### PR TITLE
Add security contact

### LINKED ISSUE #11045: Potential security issue
Hey there!

I belong to an open source security research community, and a member (@oivrip) has found an issue, but doesn’t know the best way to disclose it.

If not a hassle, might you kindly add a `SECURITY.md` file with an email, or another contact method? GitHub [recommends](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) this best practice to ensure security issues are responsibly disclosed, and it would serve as a simple instruction for security researchers in the future.

Thank you for your consideration, and I look forward to hearing from you!

(cc @huntr-helper)

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Part of #11045

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

**Review by ananthsub — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1 @@
+will@grid.ai
```
Comment (ananthsub) [inline SECURITY.md:1]: is there a general support email for the lightning project that can be added as well?

Comment (Borda) [reply SECURITY.md:1]: I think that it shall go to @williamFalcon and he can forward action to the team, but the info can be sensitive so rather not share in general email...

**Review by kaushikb11 — CHANGES_REQUESTED** [review]
Should we add `dev@pytorchlighting.ai` as well? As it is regarding security and could be high priority.

**Comment by Borda** [comment]
> Should we add `dev@pytorchlighting.ai` as well? As it is regarding security and could be high priority.

already replied here https://github.com/PyTorchLightning/pytorch-lightning/pull/11046#discussion_r768245001

File: SECURITY.md (hunk 1 above)
Comment (justusschock) [reply SECURITY.md:1]: I think that the developers should be included directly as potential security fixes would have to be fixed asap.

Comment (Borda) [reply SECURITY.md:1]:
```suggestion
Security@grid.ai
```

**Comment by williamFalcon** [comment]
@Borda 
use: developer@grid.ai and developer@pytorchlightning.ai

**Review by rohitgr7 — APPROVED** [review]

**Review by justusschock — APPROVED** [review]

**Review by rohitgr7 — COMMENTED** [review]
need a similar update in other packages as well? @Borda

**Comment by Borda** [comment]
> need a similar update in other packages as well? @Borda

good point, let's do it...

**Pull request merged**